tests/support/confirmWalletTransaction.ts

```diff
@@ -0,0 +1,42 @@
+import { Page } from '@playwright/test';
+
+import { SelectorsEnum } from './testdata';
+import { getTestIdSelector } from './testIdSelector';
+
+const confirmWithKeystore = async (walletPage: Page, password: string) => {
+  await walletPage.getByTestId(SelectorsEnum.passwordInput).fill(password);
+  await walletPage.getByTestId(SelectorsEnum.submitButton).click();
+};
+
+const confirmWithPem = async (walletPage: Page, pemPath: string) => {
+  await walletPage.setInputFiles(
+    getTestIdSelector(SelectorsEnum.walletFile),
+    pemPath
+  );
+  await walletPage.getByTestId(SelectorsEnum.submitButton).click();
+};
+
+export const confirmWalletTransaction = async (
+  page: Page,
+  loginMethod: {
```

Comment:
Let's use here the `LoginMethodType` interface you just declared in the `types.ts` file.